Plutarch.hs

```diff
@@ -22,9 +22,6 @@ module Plutarch (
   PI.punsafeConstant,
   PI.Term,
   PI.TermCont (..),
-  PL.plift,
```

Comment:
I did this because I don't see why 2 modules should export the same function. These should arguably be in the Prelude, but I saved that for another PR.